dataset/column.js

```diff
@@ -0,0 +1,217 @@
+/*
+ * column abstracts the functionality of each column
+ * of a dataset. A column has a type (text|number|date).
+ *
+ * API:
+ *
+ * column.name() ... returns the name (string)
+ * column.type() ... return column type (string)
+ * column.length ... number of rows (number)
+ * column.val(i) ... parsed value in row i
+ * column.each(func) ... apply function to each value
+ * column.raw() ... access raw, unparsed values
+ *
+ */
+
+import _ from 'underscore';
+import columnTypes from './columnTypes';
+import purifyHtml from '../../shared/purifyHtml';
```

Comment:
Tests fail because of wrong import path.
```suggestion
import purifyHtml from '../purifyHtml';
```

Reply:
> Don't know if it's worth it but in some files we could only import the underscore functions we need instead of everything.

I tried `underscore-es` which worked fine but nobody is maintaining the project anymore. Then I tried lodash but if I remember correctly the final build ended up being larger than with just underscore (even when using only lodash "submodules"). so in the end I went back to underscore. the smarter thing to do would be to replace most of the code with ES6 but that's a bigger task for later

Reply:
I think there is `lodash-es` but I agree, most could be refactored with modern JS. 👍

Reply:
ah, that might be better. it's definitely a medium to long-term goal to get rid of the jquery and underscore dependecies from any core libraries. but for now we can leave it as it is I think.